onnxruntime/core/session/onnxruntime_c_api.cc

```diff
@@ -615,6 +638,37 @@ ORT_API_STATUS_IMPL(OrtApis::RegisterCustomOpsLibrary, _Inout_ OrtSessionOptions
   API_IMPL_END
 }
 
+ORT_API_STATUS_IMPL(OrtApis::RegisterCustomOpsLibrary_V2, _Inout_ OrtSessionOptions* options,
+                    _In_ const char* library_name) {
```

Comment:
> const char* [](http://example.com/codeflow?start=24&length=12)

We need to change the type here. const ORT_TSTR* so UTF-16 can be loaded on Windows. Or UTF-8 elsewhere.

Reply:
Updated